docs/program/sdks/_index.md

```diff
@@ -1,355 +1,15 @@
 ---
-title: "Program your Robot with Viam's SDKs"
-linkTitle: "Viam SDKs"
-weight: 40
+title: "Interact with Resources with Viam's SDKs"
```

Comment:
I'm not sure about this change. I think we need to ensure program is used for seo purposes. But I'd like to hear your thoughts on why interact